build.sh

```
@@ -1,19 +1,24 @@
-#!/bin/bash
+// build.js (Windows + cross-platform equivalent of build.sh)
```

Shellcheck (0.11.0)

[error] Was this intended as a comment? Use # in sh.

(SC1127)